acceptance/acceptance_test.go

```diff
@@ -118,14 +120,12 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 	uvCache := getUVDefaultCacheDir(t)
 	t.Setenv("UV_CACHE_DIR", uvCache)
 
-	ctx := context.Background()
 	cloudEnv := os.Getenv("CLOUD_ENV")
 
 	if cloudEnv == "" {
 		defaultServer := testserver.New(t)
 		AddHandlers(defaultServer)
-		// Redirect API access to local server:
-		t.Setenv("DATABRICKS_HOST", defaultServer.URL)
+		t.Setenv("DATABRICKS_DEFAULT_HOST", defaultServer.URL)
```

Comment:
Is this the best way to pass this information to the test runs? We should consider introducing a new struct that encapsulates all the shared infrastructure. Something like:

```
type accContext struct {
  coverDir string
  defaultRepls []Replacements
  defaultServer testserver.Server
}

Reply:
This is not just for the test runner but also for test themselves (although none currently use that).